runhouse/main.py

```diff
@@ -784,12 +795,24 @@ def restart(
     )
 
 
-@app.command()
+@server_app.command()
 def stop(
+    name: Optional[str] = typer.Option(
+        None, help="A *saved* remote cluster object to restart."
+    ),
     stop_ray: bool = typer.Option(False, help="Stop the Ray runtime"),
     cleanup_actors: bool = typer.Option(True, help="Kill all Ray actors"),
 ):
+    """
+    Stopping Runhouse server on the current cluster.
```

Comment:
for consistency with the other server cmds:
```suggestion
    Stop the Runhouse server on the current cluster.
```